src/agent/src/linux_abi.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2019 Ant Financial
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Linux ABI related constants.
+
+pub const SYSFS_DIR: &str = "/sys";
+
+pub const SYSFS_PCI_BUS_PREFIX: &str = "/sys/bus/pci/devices";
+pub const SYSFS_PCI_BUS_RESCAN_FILE: &str = "/sys/bus/pci/rescan";
+#[cfg(any(
+    target_arch = "x86_64",
+    target_arch = "x86",
```

Comment:
Nit: would be good if this list was sorted alphabetically.

Reply:
fixed.